src/react/workflow/ReplicationForm.tsx

```diff
@@ -304,11 +327,15 @@ const RenderDestination = ({
         <Controller
           control={control}
           name={name}
-          render={({ field: { onChange, value: destinationLocations } }) => {
+          render={({
+            field: { onChange, onBlur, value: destinationLocations },
+          }) => {
             if (!Array.isArray(destinationLocations)) return null;
             return destinationLocations.map((destLoc: string, index) => {
               const options = destinationOptions(locations);
-              const err = errors[`${prefix}destinationLocation.${index}`];
+              const prefix_ = `${prefix}destinationLocation.${index}`;
```

Comment:
nit: is `prefix_` a specific nomenclature you want to use here or a fat-finger typo ?

Reply:
Was a specific nomenclature, changed name. (It's a standard way of doing in Haskell / PureScript / Elm world, an old habit 😅)